models/prices/optimism/prices_optimism_tokens.sql

```diff
@@ -17,6 +17,7 @@ FROM
     VALUES
     
     --tokens not yet supported or are not active on coinpaprika are commented out
+    ("op-optimism", "optimism", "OP", "0x4200000000000000000000000000000000000042", 18),
```

Comment:
this should be covered [here](https://github.com/duneanalytics/spellbook/blob/main/models/prices/prices_native_tokens.sql#L43)

Reply:
noticed your airdrop is blocked by this -- can you confirm this comment and remove if agreed?

Reply:
Oh sorry I missed your comment, OP isn't really much of a native token afaik but sure, I just removed it from my PR!

Reply:
probably makes more sense in this prices_optimism one imo. I understand 'native tokens' as tokens which don't have an address (i.e. ETH).

Reply:
okay, will move prior to merge, thanks both